src/models/FGAerodynamics.cpp

```diff
@@ -197,6 +197,8 @@ bool FGAerodynamics::Run(bool Holding)
   if (forceAxisType == atStability || momentAxisType == atStability)
 	  BuildStabilityToBodyWindAxesTransforms(in.Alpha, in.Beta, Ts2b, Ts2w);
 
+  BuildBodyToStabilityAxesTransform(in.Alpha);
```

Comment:
Why are you adding a new method ? ```BuildStabilityToBodyWindAxesTransforms``` and ```BuildBodyToStabilityAxesTransform``` could be merged in one single method.

Also do not pass ```in.Alpha``` as a parameter since it is a member of ```FGAerodynamics```.